tensorflow/compiler/mlir/hlo/lib/Dialect/mhlo/transforms/lhlo_legalize_to_affine.cc

```diff
@@ -176,6 +176,297 @@ struct ConcatOpConverter : public OpRewritePattern<ConcatenateOp> {
   }
 };
 
+/// Returns a zero value of type `eltType`. `eltType` is expected to be either
+/// int or float.
+static Value getZeroValue(Type eltType, Location loc,
```

Comment:
Nit: `eltType` name will be confusing in this context since there is no tensor here. Just `type` will be enough.

-> of type `type` which is expected to be either `int` or `float`.